appcelerator-archive/amp
# updated CLI smoke tests

### To Test
- Run `TESTINCLUDE=platform/testing testrunner platform/tests`
or
- Check Travis build
